New PHPCS failure caused by updated Drupal coding standards

## Problem/Motivation
All Quickstart builds created from the main branch are currently affected by a new PHPCS failure that is caused by a change to the Drupal PHPCS sniffs provided by the recently updated contrib Drupal Coder module.

### Describe the bug
```
FILE: ...b/profiles/custom/az_quickstart/modules/custom/az_event/az_event.views.inc
--------------------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
--------------------------------------------------------------------------------
 20 | ERROR | String concat is not required here; use a single string instead
    |       | (Generic.Strings.UnnecessaryStringConcat.Found)
--------------------------------------------------------------------------------

Time: 329ms; Memory: 40MB

Error: Process completed with exit code 1.
```

## Proposed resolution
Address the PHPCS error.

--- a/modules/custom/az_event/az_event.views.inc
+++ b/modules/custom/az_event/az_event.views.inc
@@ -17,7 +17,7 @@ function az_event_field_views_data_alter(array &$data, FieldStorageConfigInterfa
     && $data[$table_name][$field_storage->getName() . '_value']['filter']['id'] === 'date') {
       // Create year, month, and day arguments.
       $group = $data[$table_name][$field_storage->getName() . '_value']['group'];
-      $data[$table_name][$field_storage->getName() . '_value_' . 'az_calendar'] = [
+      $data[$table_name][$field_storage->getName() . '_value_az_calendar'] = [
         'title' => 'Calendar Filter ' . $field_storage->getLabel(),
         'help' => t('Calendar Filter Widget.'),
         'filter' => [

